babeld/src/ubus.c

```diff
@@ -354,6 +358,63 @@ static bool babeld_ubus_init(void) {
   return true;
 }
 
+void ubus_notify_route(struct babel_route *route, int kind) {
+  struct blob_buf b = {0};
+  char method[13]; // max is route.change
```

Comment:
char method[sizeof("route.change")]; // possible methods are route.change, route.add, route.flush

Idem below

Reply:
Or just use a largish value and don't bother (50, 100)?

Reply:
I will just go for 50. ;)